bin/custom/questions/resources/questions-spring.xml

```diff
@@ -2,38 +2,62 @@
-		<property name="supportedChannels">
-			<list value-type="de.hybris.platform.commerceservices.enums.SiteChannel">
-				<value>B2C</value>
-				<!--
-								<value>B2B</value>
-				-->
-			</list>
-		</property>
-		<property name="addonExtensionMetadata">
-			<bean class="de.hybris.platform.addonsupport.data.AddonExtensionMetadata">
-				<property name="suffixChannel" value="false"/>
-				<property name="baseExtensionName" value="questions"/>
-			</bean>
-		</property>
-	</bean>
-
-	<bean id="questionsSampleDataEventListener" parent="abstractGenericAddOnSampleDataEventListener"  >
-		<property name="extensionName" value="questions" />
-	</bean>
-
+    <context:annotation-config/>
+    <bean id="questionsCoreDataImportedEventListener" parent="abstractCoreDataImportedEventListener">
+        <property name="supportedChannels">
+            <list value-type="de.hybris.platform.commerceservices.enums.SiteChannel">
+                <value>B2C</value>
+            </list>
+        </property>
+        <property name="addonExtensionMetadata">
+            <bean class="de.hybris.platform.addonsupport.data.AddonExtensionMetadata">
+                <property name="suffixChannel" value="false"/>
+                <property name="baseExtensionName" value="questions"/>
+            </bean>
+        </property>
+    </bean>
+    <bean id="questionsSampleDataEventListener" parent="abstractGenericAddOnSampleDataEventListener">
+        <property name="extensionName" value="questions"/>
+    </bean>
+    <alias name="defaultQuestionDataPopulator" alias="questionDataPopulator"/>
+    <bean id="defaultQuestionDataPopulator"
+          class="com.myapp.questions.populators.QuestionDataPopulator">
+    </bean>
+    <alias name="defaultQuestionConverter" alias="questionConverter"/>
+    <bean id="defaultQuestionConverter" parent="abstractPopulatingConverter">
+        <property name="targetClass" value="com.myapp.questions.data.QuestionData"/>
+        <property name="populators">
+            <list>
+                <ref bean="questionDataPopulator"/>
+            </list>
+        </property>
+    </bean>
+    <alias name="defaultProductQuestionDataPopulator" alias="productQuestionDataPopulator"/>
+    <bean id="defaultProductQuestionDataPopulator"
+          class="com.myapp.questions.populators.ProductQuestionDataPopulator">
+        <property name="questionConverter" ref="questionConverter"/>
+    </bean>
+    <bean id="productConverter" parent="abstractPopulatingConverter">
+        <property name="targetClass" value="de.hybris.platform.commercefacades.product.data.ProductData"/>
+        <property name="populators">
+            <list>
+                <ref bean="productQuestionDataPopulator"/>
+            </list>
+        </property>
+    </bean>
+    <alias name="defaultProductFacade" alias="productFacade"/>
+    <bean id="defaultProducerFacade" class="com.myapp.questions.facades.impl.DefaultProductFacade">
```

Comment:
defaultProductFacade. Additionally, alias name and bean id have different values - is it a typo?